tutorial/10_trame/README.rst

```diff
@@ -17,11 +55,57 @@ visualization in reactive web applications.
 
 High-level framework 1 + high-level framework 2 = a streamlined approach to making powerful web applications with 3D visualization front and center.
 
+The following code creates a simple Trame application with a PyVista plotter embedded in the UI.
+This code can be used as a base for building all of your Trame applications with PyVista as it
+contains all of the boilerplate code needed to get started.
+
+
+.. tab-set::
+
+   .. tab-item:: Code
+
+      .. code:: python
+
+        from trame.app import get_server
+        from trame.ui.vuetify import SinglePageLayout
+
+        import pyvista as pv
+        from pyvista.trame.ui import plotter_ui
+
+        # Always set PyVista to plot off screen with Trame
+        pv.OFF_SCREEN = True
+
+        server = get_server()
+        state, ctrl = server.state, server.controller
+
+        mesh = pv.Wavelet()
+
+        pl = pv.Plotter()
+        pl.add_mesh(mesh)
+
+        with SinglePageLayout(server) as layout:
+            with layout.content:
+                # Use PyVista's Trame UI helper method
+                #  this will add UI controls
+                view = plotter_ui(pl)
```

Comment:
@banesullivan  For the Trame beginner, the other parts are intuitive, but using with is a little more thoughtful. I would like to see a detailed explanation of why. Is it for memory resource management?